### PR TITLE
[MODORDERS-1166] Add bindItemTenantId to Piece schema

### PR DESCRIPTION
**Pull request by Saba-Zedginidze-EPAM**
## Purpose
[[MODORDERS-1166] Add bindItemTenantId to Piece schema](https://folio-org.atlassian.net/browse/MODORDERS-1166)

## Approach
- Add bindItemTenantId to Piece

### CODE REVIEW

File: mod-orders-storage/schemas/piece.json [hunk 1]
```diff
@@ -49,6 +49,10 @@
       "description": "Receiving tenant for ECS-enabled clusters",
       "type": "string"
     },
+    "bindItemTenantId": {
+      "description": "Bind item tenant for ECS-enabled clusters",
+      "type": "string"
+    },
```
Comment (SerhiiNosko) [inline mod-orders-storage/schemas/piece.json:55]: lets move near bindItemId

Comment (SerhiiNosko) [reply mod-orders-storage/schemas/piece.json:55]: I think it is our last schema change in this release. Probably let's update all acq modules with schema change update

**Review by SerhiiNosko — APPROVED** [review]

File: mod-orders-storage/schemas/piece.json (hunk 1 above)
Comment (Saba-Zedginidze-EPAM) [reply mod-orders-storage/schemas/piece.json]: Moved under bindItemId

Comment (Saba-Zedginidze-EPAM) [reply mod-orders-storage/schemas/piece.json]: Will update all modules once this is merged then

**Pull request merged**